vulcan/lib/server/workflows/umap.cwl

```diff
@@ -223,7 +229,20 @@ steps:
   downloadRawData:
     run: ui-outputs/link.cwl
     in:
-      a: calc_umap/umap_anndata.h5ad
+      a: calc_leiden/leiden_anndata.h5ad
```

Comment:
Connected with my last comment, I've re-wired the download button to output the leiden_anndata output.